Content.Shared/GameObjects/EntitySystems/ActionBlocker/ActionBlockerSystem.cs

```diff
@@ -36,8 +43,13 @@ public static bool CanInteract(IEntity entity)
             return canInteract;
         }
 
-        public static bool CanUse(IEntity entity)
+        public static bool CanUse([NotNullWhen(true)] IEntity? entity)
```

Comment:
[NotNullWhen(true)] seems iffy?

Reply:
I think this is okay, marking as resolved for now.